moml/simulation/molecular_dynamics/force_field/plugins/ix_sdb_v1/build.py

```
@@ -0,0 +1,22 @@
+
+def build(tmp_dir: Path, cfg: dict) -> Tuple[Path, Topology, List[int]]:
+    bead_count      = cfg["bead_count"]
+    bead_radius_nm  = cfg["bead_radius_nm"]
+
+    # 1. Load styrene-divinylbenzene-backbone monomer with quaternary amine
+    monomer = Molecule.from_smiles("[CH2]C(=O)O[C@H]1CC[N+](C)(C)C1")  # placeholder
+    # 2. Pack monomers randomly inside a sphere with Packmol, polymerise manually
+    positions, topology = _pack_bead(monomer, bead_radius_nm, bead_count)
+
+    pdb_out = tmp_dir / "ix_bead.pdb"
+    PDBFile.writeFile(topology, positions, open(pdb_out, "w"))
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)